packages/next/README.md

```diff
@@ -1781,6 +1763,14 @@ module.exports = {
 }
 ```
 
+## Automatic Pre-rendering
+
+Next.js will automatically choose between a `dynamic` or `static` page at build time with `next build`, this is based on whether or not `getInitialProps` is set in the page, if set Next.js will assume that your page is `dynamic`, or `static` otherwise.
+
+This allows Next.js to only pre-render pages that rely on dynamic data, and to export a static `html` file for non dynamic pages.
```

Comment:
pre-render means statically exporting
dynamic rendering is server-side rendering

Reply:
Right, updated. 👍